<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QStackedWidget" name="stackedWidget_2">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page_3">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="2">
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="startbon">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>70</height>
                 </size>
                </property>
                <property name="text">
                 <string>按压</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="level" native="true">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="2" column="0">
                  <widget class="QRadioButton" name="eazy">
                   <property name="text">
                    <string>eazy</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QRadioButton" name="insane">
                   <property name="text">
                    <string>Insane</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0" colspan="2">
                  <widget class="QLabel" name="levellabel">
                   <property name="text">
                    <string>请选择难度^_^</string>
                   </property>
                   <property name="margin">
                    <number>7</number>
                   </property>
                   <property name="indent">
                    <number>20</number>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QRadioButton" name="demon">
                   <property name="text">
                    <string>Demon</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QRadioButton" name="hard">
                   <property name="text">
                    <string>hard</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0" colspan="2">
                  <widget class="QLabel" name="rebonlabel">
                   <property name="text">
                    <string>重置后需要再次点击！</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="changebutton">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>70</height>
                 </size>
                </property>
                <property name="text">
                 <string>给予通关区间</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="button">
                <property name="text">
                 <string>认输</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="image.qrc">:/Photo/character.jpg</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="lcdNumber"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_4">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="overlabel">
             <property name="text">
              <string>游戏失败</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="rebon">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>重置</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="1">
        <widget class="QPushButton" name="Mainbutton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string>开始！ </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="startlabel">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>400</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="image.qrc">:/Photo/main.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
